src/styles/blocks/card.scss

```diff
@@ -0,0 +1,93 @@
+.card {
+  box-sizing: border-box;
+  padding: 32px 16px 15px;
+  width: 200px;
+  border-radius: 5px;
+  background-color: #fff;
+  color: $color-main;
+  border: 1px solid #f3f3f3;
+
+  &__image {
+    padding-inline: 3px;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), avoid using padding-inline as it is not fully supported by all browsers. Consider using `padding-left` and `padding-right` instead.